readme.md

```diff
@@ -568,6 +568,8 @@ Assert that `function` throws an error or `promise` rejects.
 
 `error` can be a constructor, regex, error message or validation function.
 
+Returns the error thrown by `function` or the reason of `promise` rejection.
```

Comment:
Much better. I struggled for a good couple of minutes during my commute home on how to word that. ugh. 

Thanks!